addons/default/visiosoft/advs-module/resources/views/list/gallery.twig

```diff
@@ -13,8 +13,16 @@
             <div class="col-xl-4 col-6 p-sm-3 p-0">
                 <div class="d-flex flex-column product-hb gallery-card pb-3 px-1 pt-1 border shadow-sm rounded">
                     <div class="product-image-hb p-sm-3 p-1">
-                        <a href="{{ adv.detail_url }}">
-                            <img class="product-image-in" src="{{ adv.cover_photo }}" alt="{{ adv.name }}">
+                        <a href="{{ adv.detail_url }}" class="imgs" id="{{ adv.id }}">
+                                {%  for key,photo in adv.getViewPhotoUrl|slice(0, 3) %}
```

Comment:
You can assign a variable instead of calling it twice

 {% set viewPhotos = adv.getViewPhotoUrl|slice(0, 3) %}

Reply:
couldn't make it. which specific line I should declare it in?

Reply:
done